packages/merge-styles/src/styleToClassName.ts

```diff
@@ -251,12 +269,10 @@ export function applyRegistration(registration: IRegistration): void {
       const rules = rulesToInsert[i + 1];
       if (rules) {
         let selector = rulesToInsert[i];
-
-        selector = selector.replace(/&/g, '.' + registration.className);
+        selector = selector.replace(/&/g, repeatString(`.${registration.className}`, specificityMultiplier));
```

Comment:
regex is expensive. maybe it should be conditionalized only when the multiplier exists.

Reply:
regex replacement logic here is the same logic before and after my change?